.ci/validation.sh

```diff
@@ -201,7 +201,6 @@ no-error-xwiki)
   cd ..
   checkout_from https://github.com/xwiki/xwiki-platform.git
   cd .ci-temp/xwiki-platform
-  git checkout "b31a47e4f84b1981c27fd55716c""eb89f6983af5b"
```

Comment:
Please recheck history of this certain hash checkout, we might setup it because xwiki team is not diligent on each commit to has zero violations.

Reply:
This hash comes from https://github.com/checkstyle/checkstyle/pull/9980

This Issue may be related: https://github.com/checkstyle/checkstyle/issues/9967
I'll add a note.